.github/actions/node/build/action.yml

```diff
@@ -0,0 +1,35 @@
+name: NodeJS build
+
+inputs:
+  NODE_VERSION:
+    required: true
+    type: string
+  RELEASE_ENVIRONMENT:
+    required: true
+    type: string
+  WORKING_DIRECTORY:
+    required: true
+    type: string
+
+runs:
+    using: composite
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js ${{ inputs.NODE_VERSION }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ inputs.NODE_VERSION }}
+          cache: 'npm'
```

Comment:
Nel file `node-step-format-lint-build.yml` vedo che è presente il parametro `cache-dependency-path: ${{ inputs.WORKING_DIRECTORY }}/package-lock.json` che ricorso essere essenziale per risolvere correttamente la cache per i progetti NX. Potresti confermare che funziona anche in quei casi?

Reply:
hai perfettamente ragione, mi era rimasto indietro. Aggiunto! :)